vocab/security/vocabulary.yml

```diff
@@ -43,6 +45,11 @@ class:
       - label: EdDSA Cryptosuite v2022
         url: https://www.w3.org/TR/vc-di-eddsa/#multikey
 
+  - id: JsonWebKey
+    label: JSON Web Key Verification Method
+    upper_value: sec:VerificationMethod
+    defined_by: https://www.w3.org/TR/vc-data-integrity/#jsonwebkey
```

Comment:
I did not see the `<dfn>` statement in the text...

Reply:
ReSpec auto-generates it based on this line: https://github.com/w3c/vc-data-integrity/pull/135/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R1094

Reply:
That is correct for the class specification. However, #148, that adds both the `secretKeyJwk` and `publicKeyJwk` properties rely on the presence of `<dfn`> for those. I see it for `publicKeyJwk` at https://pr-preview.s3.amazonaws.com/w3c/vc-data-integrity/pull/135.html#dfn-publickeyjwk, I did not find it for `secretKeyJwk` :-(

Reply:
Ah, right. I'll raise another PR to refactor the sections and put the definitions where they belong (in the sections defining both the `JsonWebKey` type and the `Multikey` type.

Reply:
PR #152 has been raised to address this issue.